File: Results/DER/Translation/translation_output/misleading_test/mistral/codenet/python/java/p02627.java
<s> [INST] Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
B

Expected output:
A

```
n = input()

if n.isupper():
    print("A")
elif n.islower():
    print("a")
```
[/INST]
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String n = scanner.nextLine();

        if (n.isUpperCase()) {
            System.out.println("A");
        } else if (n.isLowerCase()) {
            System.out.println("a");
        }
    }
}
```</s>

File: Results/DER/Translation/translation_output/misleading_test/mistral/codenet/java/python/p02627.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
B

Expected output:
A

```
import java.util.*;

public class p02627{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        System.out.println(str.matches("[A-Z]") ? "A" : "a");
    }
}
```
[/INST]
```python
str = input()
if str.isalpha():
    print("A")
else:
    print("a")
```</s>
